CONTRIBUTING.md

```diff
@@ -91,6 +91,21 @@ yarn react-native run-ios --mode=Release
 yarn react-native run-android --mode=Release
 ```
 
+### Optional
+
+You can optionally start the Metro bundler if you want to control where it runs:
+
+```sh
+yarn start --reset-cache
+```
+
+Run the emulators:
+
+```sh
+yarn react-native run-ios
+yarn react-native run-android
+```
+
```

Comment:
Thanks, probably a bad merge on my side.

Actually I think we could remove this section completely.

As this section in the document describes the exact same https://github.com/getsentry/sentry-react-native/pull/4140/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R51-R59